src/services/conversation.service.ts

```diff
@@ -7,22 +7,14 @@ import { Conversation } from '@entities/conversation.entity';
 export class ConversationsService {
```

Comment:
I know that is not part of your code but, boy scout rule, with each commit, you leave the code better than you found it. Just a friendly advice (:

Reply:
It is part of my code, but I didn't do PRs for previous commits. Later I will refactor the code considering all your feedback